www/src/data/sidebars/doc-links.yaml

```diff
@@ -1,729 +0,0 @@
-- title: Documentation
```

Comment:
I this intentional? You can restore it by doing `git checkout master -- www/src/data/sidebars/doc-links.yaml